dm/worker/task_checker.go

```diff
@@ -263,25 +263,21 @@ func isResumableError(err *pb.ProcessError) bool {
 		return true
 	}
 
-	switch err.Type {
-	case pb.ErrorType_ExecSQL:
```

Comment:
We should remove this `ProcessError.Type` later because we have better `code`, `class`, etc. cc @GMHDBJD

Reply:
Will do it in #746 later